analysis/forecast_plots.py

```diff
@@ -142,6 +144,14 @@ def read_in_Reff(forecast_R=None,R_I=None,file_date = "2020-04-01"):
    
 
             df_forecast = df_forecast.loc[df_forecast.type==forecast_R]
+
+            if VoC_flag != '':
+                # Here we apply the same beta(6,14)+1 scaling from VoC to the Reff data for plotting
+                # We do so but editing a slice of the data frame. Forgive me for my sins.
+                VoC_dates_to_apply_idx = df_forecast.index[pd.to_datetime(df_forecast.date, format='%Y-%m-%d') > pd.to_datetime(file_date)]
+                df_slice_after_VoC = df_forecast.iloc[VoC_dates_to_apply_idx, 8:]
+                df_forecast.iloc[VoC_dates_to_apply_idx , 8:] = df_slice_after_VoC*(beta.rvs(6,14, size = df_slice_after_VoC.shape) + 1)
```

Comment:
Why are the 8th column? Might want to comment in that for future Tobin (or Tobin-replacement :) )

Reply:
Comments added. Future Tobin sends his thanks :)